File: easy/pascal/onboarding.pas
program Game;

var
  enemy1, enemy2: string;
  dist1, dist2: integer;

begin
  while true do
  begin
    readln(enemy1);
    readln(dist1);
    readln(enemy2);
    readln(dist2);

    if dist1 < dist2 then
      writeln(enemy1)
    else
      writeln(enemy2);
  end;
end.

File: easy/php/onboarding.php
<?php

while (true) {
    $enemy1 = trim(fgets(STDIN));
    $dist1 = (int)trim(fgets(STDIN));
    $enemy2 = trim(fgets(STDIN));
    $dist2 = (int)trim(fgets(STDIN));

    if ($dist1 < $dist2) {
        echo $enemy1 . PHP_EOL;
    } else {
        echo $enemy2 . PHP_EOL;
    }
}

?>
